### ISSUE TITLE
Waves Soundgrid Studio

### ISSUE THREAD

**Issue by CalvarySteve** [bitfocus/companion-module-requests#1389]
Will Bitfocus Companion ever explore Waves Software and being able to control it? Such as Soundgrid Studio? Thanks for the consideration

**Comment by DrumFunKen** [comment]
Yes we need someone to finally make this happen. Basically it's just midi commands. If there was a generic Midi program for companion that would be a great start. 
https://www.waves.com/mixers-racks/superrack
https://www.waves.com/mixers-racks/superrack-performer

**Comment by phillipivan** [comment]
@DrumFunKen like [this](https://github.com/bitfocus/companion-module-generic-midi)?

**Comment by DrumFunKen** [comment]
I wish that populated in my companion GUI i have no idea how to load it from github.

**Comment by phillipivan** [comment]
It doesn't have a release published yet. The [companion-module-base wiki](https://github.com/bitfocus/companion-module-base/wiki) has details on how to setup a dev environment and build a module. With all of that said, I have no idea what state the module is in, its obviously quite new.

**Comment by philipcain** [comment]
MIDI will only get you so far in the Waves Soundgrid apps (Studio, SuperRack, LV1). They have undocumented remote protocols that would be amazing to see implemented in Companion.

**Comment by isaacdantzler** [comment]
I've wondered if there would be a way to reverse engineer the soundgrid apps that let you control mixes from an iPad/iPhone - they seem to have full feedback and control. Would be really nice to adjust faders for some channels that way.
